packages/core/lib/commands/test/meta.js

```diff
@@ -45,10 +45,10 @@ module.exports = {
       type: "boolean",
       default: false
     },
-    reporter: {
+    "reporter": {
       alias: "r",
       describe: "Specify the type of mocha reporter",
-      default: "spec"
+      default: undefined
```

Comment:
Could you explain the reason for this change?  (I'm pretty sure I know, having made exactly this sort of mistake before... but I imagine most people won't find this obvious, I think this could do with some explanation.)

Reply:
This doesn't actually make much difference though. As undefined also will display the "spec" type of mocha reporter.

Reply:
Uh, apologies, but that doesn't seem like an answer to what I asked...?

(I'm not saying this change is wrong -- I'm pretty sure it's right!  But it's not *obvious*, and you didn't explain it, so I want the explanation written down to ensure that everyone understands.)

Reply:
I have made the default as "undefined" because of the fact that the value of the reporter will be undefined if the user doesn't specify any specific reporter as the command line argument and also in the config. And in that case, the default mocha reporter "spec" will be used. However, both behaves in the same way. So, instead of specifying "spec" as the default, i preferred using "undefined" as the default.

Reply:
Oh, so it's just a style thing?  Interesting, that's different from what I had thought -- I had assumed that you changed it because using `"spec"` would cause it to accidentally use `"spec"` in cases where it shouldn't.  I caused a problem like that a while back, and assumed this was similar.  Glad you explained it, because I evidently misunderstood!